src/cmd/tools/wait.go

```diff
@@ -29,6 +30,7 @@ var waitForCmd = &cobra.Command{
 	Example: lang.CmdToolsWaitForExample,
 	Args:    cobra.MinimumNArgs(2),
 	Run: func(cmd *cobra.Command, args []string) {
+		exec.ExitOnInterrupt(1, lang.ErrInterrupt)
```

Comment:
Can we make this call in the `rootCmd` `PersistentPreRun` to handle all exits for Zarf commands (after the check for vendored commands)?

https://github.com/defenseunicorns/zarf/blob/4d4ae8fe077b03ef06f2674cdcb5d310c04bb125/src/cmd/root.go#L39

That would allow it to work for this command and be a good starting point for https://github.com/defenseunicorns/zarf/issues/1518